### PR TITLE
demonstrated an use of NSFetchedResultsControllerDelegate Protocol in Re...

### PR DESCRIPTION
**Pull request by naoyamakino**
...cipe app to update table view once a recipe is added.
see http://developer.apple.com/library/ios/#documentation/CoreData/Reference/NSFetchedResultsControllerDelegate_Protocol/Reference/Reference.html

### CODE REVIEW

File: examples/Recipes/app/controllers/recipe_list_table_view_controller.rb [hunk 1]
```diff
@@ -52,6 +52,47 @@ def fetchedResultsController
...
+  def controller(controller, didChangeObject:anObject, atIndexPath:indexPath, forChangeType:type, newIndexPath:newIndexPath)
+    tableView = self.tableView
+    case type
+    when NSFetchedResultsChangeInsert
+      then tableView.insertRowsAtIndexPaths(NSArray.arrayWithObject(newIndexPath),
+                                            withRowAnimation:UITableViewRowAnimationFade)
+    when NSFetchedResultsChangeDelete
+      then tableView.deleteRowsAtIndexPaths(NSArray.arrayWithObject(indexPath),
+                                            withRowAnimation:UITableViewRowAnimationFade)
+      #TODO case for NSFetchedResultsChangeUpdate is complaining that self does not have configureCell:atIndexPath method.
```
Comment (alloy) [inline examples/Recipes/app/controllers/recipe_list_table_view_controller.rb]: The `configureCell:` method is just an abstracted method that prepares cells which is in their example **only**, it’s not part of the protocol. I’m not sure where I saw the full example code in the past, but the implementation basically does what you normally do in `tableView:cellForRowAtIndexPath:`, so you should be able to figure that out yourself.

**Comment by alloy** [comment]
Great work, @naoyamakino! Once you’ve cleaned up the code with my inline comment, we can merge this in.

**Comment by naoyamakino** [comment]
@alloy thanks for the comment, made the change.

**Comment by alloy** [comment]
Awesome, thanks!

**Pull request merged**